site/src/Entity/User.php

```diff
@@ -134,6 +134,14 @@ class User extends BaseUser implements NotifiableInterface
      */
     protected $updatedAt;
 
+    /**
+     * Has the user subscribed to email notifications?
+     *
+     * @var $subscribedToEmailNotifs
+     *
+     * @ORM\Column(type="boolean", options={"default" : true})
+     */
+    protected $subscribedToEmailNotifs = true;
```

Comment:
Notifs => Notifications ?
Better to use the full spelling

Reply:
Okay